bin/dry-run.js

```diff
@@ -209,3 +210,14 @@ async function fetchMeasurementsAddedFromChain (roundIndex) {
 
   return events.filter(e => e.roundIndex.eq(roundIndex)).map(e => e.cid)
 }
+
+function createPgClient () {
```

Comment:
why is it ok to pass a mock here?

Reply:
I don't expect dry runs to modify any data. spark-evaluate uses the PG connection to update `spark_stats` database. I am using a mock because I don't want to do that. (And 

I also want a simpler setup that does not require the person running a dry run to have PG up & running and/or DATABASE_URL correctly configured.

Reply:
Ok! Could you please add this as a comment here? Or, could we pass a function that mocks less? Assuming no queries will happen. I think it should throw if it tries to query the noop mock

Reply:
> Could you please add this as a comment here?

Done: 70ee0e4

> Or, could we pass a function that mocks less? Assuming no queries will happen. I think it should throw if it tries to query the noop mock

`evaluate()` is expected to run SQL queries to update the statistics. 

I am afraid we don't have a different function that we could execute for the dry run if we want the dry run to report all information produced by the real run - e.g. what scores we submit on the chain, what telemetry we report to InfluxDB and so on.

We are already passing a stub function for `recordTelemetry`. So at least, we are staying consistent in this PR.

Reply:
Sure, done in [7b451bb](https://github.com/filecoin-station/spark-evaluate/pull/125/commits/7b451bb4e13032a691d378e2ad164501fda19720).